apps/web/app/agenda/page-new.tsx

```
@@ -0,0 +1,1633 @@
+}: {
+  appointments: Appointment[];
+  selectedDate: Date;
+  onDateSelect: (date: Date) => void;
+  viewMode: "month" | "week" | "day";
+  onAppointmentClick: (appointment: Appointment) => void;
+}) {
+  const getDaysInView = () => {
+    switch (viewMode) {
+      case "month":
+        const start = startOfMonth(selectedDate);
+        const end = endOfMonth(selectedDate);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)